YelpAPI.podspec

```diff
@@ -29,4 +29,5 @@ Pod::Spec.new do |s|
 
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'TDOAuth', '~> 1.1'
+  s.private_header_files = 'Classes/Client/YLPClientPrivate.h', 'Classes/Common/YLPCommonPrivate.h', 'Classes/Response/YLPResponsePrivate.h'
```

Comment:
This could be `'Classes/**/*Private.h'`; I kind of like that because if we add any more private headers they'll get picked up automatically (assuming we follow the same convention)